unit_tests/test_alternator_streams_kcl.py

```diff
@@ -0,0 +1,68 @@
+import time
+import pytest
+import logging
+
+import sdcm.utils.alternator as alternator
+from sdcm.ycsb_thread import YcsbStressThread
+from sdcm.kcl_thread import KclStressThread
+from unit_tests.dummy_remote import LocalLoaderSetDummy
+
+pytestmark = [pytest.mark.usefixtures('events'),
+              pytest.mark.skipif(True, reason="those are integration tests only")]
+
+ALTERNATOR_PORT = 8000
+TEST_PARAMS = dict(dynamodb_primarykey_type='HASH',
+                   alternator_use_dns_routing=True, alternator_port=ALTERNATOR_PORT)
+ALTERNATOR = alternator.api.Alternator(sct_params={"alternator_access_key_id": None,
+                                                   "alternator_secret_access_key": None,
+                                                   "alternator_port": ALTERNATOR_PORT})
+
+
+def test_01_kcl_with_ycsb(request, docker_scylla, events):
+    loader_set = LocalLoaderSetDummy()
+    num_of_keys = 1000
+    # 1. start kcl thread and ycsb at the same time
+    ycsb_cmd = f'bin/ycsb load dynamodb  -P workloads/workloada -p recordcount={num_of_keys} -p dataintegrity=true -p insertorder=uniform -p insertcount={num_of_keys} -p fieldcount=2 -p fieldlength=5'
+    ycsb_thread = YcsbStressThread(loader_set, ycsb_cmd, node_list=[docker_scylla], timeout=600, params=TEST_PARAMS)
+
+    kcl_cmd = f"hydra-kcl -t usertable -k {num_of_keys}"
+    kcl_thread = KclStressThread(loader_set, kcl_cmd, node_list=[docker_scylla], timeout=600, params=TEST_PARAMS)
+
+    def cleanup_thread():
+        ycsb_thread.kill()
+        kcl_thread.kill()
+
+    request.addfinalizer(cleanup_thread)
+
+    time.sleep(60)
+    kcl_thread.run()
+    time.sleep(30)
```

Comment:
what are these sleeps needed for?

Reply:
to give the kcl_thread chance to create the tables, before ycsb starts starts.

FYI, this integration test, was mainly a helper to build/test the code locally, while writing it. (we are not yet running them regularly anywhere)